README.Rmd

```diff
@@ -98,7 +98,7 @@ visualize(null_distn) +
   shade_p_value(obs_stat = F_hat, direction = "greater")
 ```
 
-```{r viz-graphic, message = FALSE, warning = FALSE, echo = FALSE}
+```{r viz-graphic, message = FALSE, warning = FALSE, echo = FALSE, fig.cap = " ", fig.alt = "A histogram showing a distribution of F statistics, right-tailed and centered around one half. The x axis ranges from zero to five. The region of the histogram to the right of the observed statistic, just above two, is shaded red to represent the p-value."}
```

Comment:
Is "one half" or "one point five" or "1.5" better? I'm actually not sure for alt text which is better.

Reply:
Me neither. I'll read up on this.